### DISCUSSION TITLE
How to not prepend website title to blog post titles in the HTML title tag?

### DISCUSSION THREAD

**Discussion by rohitfarmer** [quarto-dev/quarto-cli#6934] [Q&A] [answered]
### Description

I am trying to figure out how to **NOT** prepend the website title to the blog post titles in the HTML <title> tag. Prepending the main website title to the blog post title is making the HTML page title too long for SEO. I want the main website title only for the homepage and maybe for the About and blog post listing pages, but not on every page. Especially not in individual blog posts. 

Any suggestions would help. thanks!

**Comment by mcanouil** [comment]
Use `pagetitle`, see <https://quarto.org/docs/reference/formats/html.html#metadata>.

**Reply by rohitfarmer** [reply]
Thank you. I found this option, and it works. But this fix would mean I will have two lines with the same title in the blog post-YAML, which is repetitive (below), and I will make a mistake. Is there a way to implement it more globally for all the qmds within a folder, or can I pass the `title` to `pagetitle` via variable?

```
title: "Deep Creek Lake State Park Campground, MD, USA"
pagetitle: "Deep Creek Lake State Park Campground, MD, USA"
```

**Reply by mcanouil** [reply]
Uee the meta shortcode (<https://quarto.org/docs/authoring/variables.html#meta>)

```yml
title: "Deep Creek Lake State Park Campground, MD, USA"
pagetitle: "{{< meta title >}}"
```

And if you don't want to bother typing it in your blog posts, use the `_metadata.yml` file (see <https://quarto.org/docs/projects/quarto-projects.html#shared-metadata>) in your blog post directory to set `pagetitle: "{{< meta title >}}"`.

**Reply by rohitfarmer** [reply]
Sorry for a slightly different question in this thread, but it's related to the shortcodes you mentioned. 

In the example below, I want to give a path to the header image in the YAML and then insert it at the beginning of the text. The variable pulls the image path, but it adds a ".png" extension in the end ("https://storage.googleapis.com/outdoorsy-indians-images/posts/bushcraft-shelter.jpg.png"), altering the URL. How can I fix it? Or a better method to include the header image in a non-repetitive and cleaner way.

```
---
title: "Essentials for Outdoor Survival"
pagetitle: "{{< meta title >}}"
image: https://storage.googleapis.com/outdoorsy-indians-images/posts/bushcraft-shelter.jpg
image-alt: "Photo of a bushcraft shelter"
---

![{{< meta image-alt >}}]({{< meta image >}})
```

**Reply by mcanouil** [reply, marked as answer]
I am guessing you are using the stable 1.3 version of Quarto or earlier.
This is fixed in current 1.4 pre-release.
<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/0fd992be-da81-4c9b-a4c5-f94029817a16">

**Reply by rohitfarmer** [reply]
I am using 1.3.450

**Reply by rohitfarmer** [reply]
I upgraded to the pre-release, and it worked. Thank you so much.
